connect/src/wormhole.ts

```diff
@@ -222,6 +223,38 @@ export class Wormhole<N extends Network> {
     return await tb.getOriginalAsset(token.address);
   }
 
+  /**
+   * Returns the UniversalAddress of the token. This may require fetching on-chain data.
+   * @param chain The chain to get the UniversalAddress for
+   * @param token The address to get the UniversalAddress for
+   * @returns The UniversalAddress of the token
+   */
+  async getTokenUniversalAddress<C extends Chain>(
```

Comment:
Is this function exported or maybe in the future? If so, we might want to check the params.

Reply:
I'm not sure what you mean by checking the params, do you have an example?

Reply:
I meant null-checking the params inside the function, but we may also let it throw so the caller can handle it.

Reply:
I think for now the simplest thing is to let the caller catch the exception or let it bubble up in case of an error. If there's a network error, then there's nothing we can really do